__tests__/app.test.js

```diff
@@ -0,0 +1,54 @@
+const app = require("../app")
+const seed = require("../db/seeds/seed")
+const request = require('supertest')
+const db = require("../db/connection")
+const testData = require("../db/data/test-data/index")
+
+
+beforeEach(() => {
+    return seed(testData)
+    });
+
+  afterAll(() => {
+    db.end()
+  });
+
+describe("GET API/topics", () => {
+    test("Should return an array", () => {
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({body}) => {
+            expect(body).toBeInstanceOf(Array)
+        })
+    })
+    test("Array should have objects inside of it.", () => {
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({body}) => {
+            expect(body[0]).toBeInstanceOf(Object)
```

Comment:
As we are making an assertion later regarding the keys of the elements of this array, it must be an object that comes back inside the array. Making this test redundant.